src/pages/ReimbursementAccount/PersonalInfo/substeps/Confirmation.tsx

```diff
@@ -0,0 +1,135 @@
+import React, {useMemo} from 'react';
+import {ScrollView, View} from 'react-native';
+import type {OnyxEntry} from 'react-native-onyx';
+import {withOnyx} from 'react-native-onyx';
+import Button from '@components/Button';
+import DotIndicatorMessage from '@components/DotIndicatorMessage';
+import MenuItemWithTopDescription from '@components/MenuItemWithTopDescription';
+import ScreenWrapper from '@components/ScreenWrapper';
+import Text from '@components/Text';
+import TextLink from '@components/TextLink';
+import useLocalize from '@hooks/useLocalize';
+import type {SubStepProps} from '@hooks/useSubStep/types';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as ErrorUtils from '@libs/ErrorUtils';
+import getSubstepValues from '@pages/ReimbursementAccount/utils/getSubstepValues';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type {ReimbursementAccount, ReimbursementAccountForm} from '@src/types/onyx';
+
+type ConfirmationOnyxProps = {
+    /** Reimbursement account from ONYX */
+    reimbursementAccount: OnyxEntry<ReimbursementAccount>;
+
+    /** The draft values of the bank account being setup */
+    reimbursementAccountDraft: OnyxEntry<ReimbursementAccountForm>;
+};
+
+type ConfirmationProps = ConfirmationOnyxProps & SubStepProps;
+
+const PERSONAL_INFO_STEP_KEYS = CONST.BANK_ACCOUNT.PERSONAL_INFO_STEP.INPUT_KEY;
+
+function Confirmation({reimbursementAccount, reimbursementAccountDraft, onNext, onMove}: ConfirmationProps) {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+
+    const isLoading = reimbursementAccount?.isLoading ?? false;
+    const values = useMemo(() => getSubstepValues(PERSONAL_INFO_STEP_KEYS, reimbursementAccountDraft, reimbursementAccount), [reimbursementAccount, reimbursementAccountDraft]);
+    const error = ErrorUtils.getLatestErrorMessage(reimbursementAccount ?? {});
+
+    return (
+        <ScreenWrapper
+            testID={Confirmation.displayName}
+            style={[styles.pt5]}
+        >
+            <ScrollView contentContainerStyle={styles.flexGrow1}>
+                <Text style={[styles.textHeadlineLineHeightXXL, styles.ph5, styles.mb3]}>{translate('personalInfoStep.letsDoubleCheck')}</Text>
+                <MenuItemWithTopDescription
+                    description={translate('personalInfoStep.legalName')}
+                    title={`${values[PERSONAL_INFO_STEP_KEYS.FIRST_NAME]} ${values[PERSONAL_INFO_STEP_KEYS.LAST_NAME]}`}
+                    shouldShowRightIcon
+                    onPress={() => {
+                        onMove(0);
```

Comment:
I think it's a good idea to define named constants for values passed as the `onMove` parameter instead of using magic numbers. This makes the code more readable rather than trying to figure out what `0`, `1`, `2`, etc means.